idea_town/frontend/static-src/test/lib/router.js

```diff
@@ -0,0 +1,100 @@
+import test from 'tape';
+import Router from '../../app/lib/router';
+import app from 'ampersand-app';
+import Me from '../../app/models/me';
+import Experiments from '../../app/collections/experiments';
+
+test(`🖳  Running Tests for ${__filename} ␄`, a => a.end());
```

Comment:
Seems like this bit of boilerplate could be extracted out to a runner or parent class, too.